.github/workflows/nonprod.yaml

```diff
@@ -6,12 +6,22 @@ on:
     paths-ignore:
       - "README.md"
       - "catalog.yaml"
+
+  workflow_dispatch:
+    inputs:
+      gratibot_limit:
+        description: "The amount of fistbumps that a user can give for one day."
+        required: false
+        default: "5"
+        type: string
+
 env:
   IMAGE_NAME: ghcr.io/${{ github.repository_owner }}/gratibot
 
 jobs:
   build:
     name: Build and Publish Image
+    if: github.event_name == 'push' || contains(['Pactionly', 'gesparza3'], github.actor)
```

Comment:
Will this condition still be in effect during normal pushes to `main`?

Reply:
yes, the condition is true if the event is a push to main or a workflow_dispatch trigger by you or Alice